.travis.yml

```diff
@@ -6,11 +6,17 @@ dist: trusty
 install: 
     - sudo pip install mkdocs
     - sudo pip install mkdocs-material
-script: 
-    - ./gradlew buildJar buildWar buildCli buildDoc -PbuildNumber=${TRAVIS_JOB_NUMBER} ${CLIENT_PATH} && ./e2e/integration_tests.sh
+script:
+    - ./gradlew buildDoc buildJar buildWar buildCli buildDocker -PbuildNumber=${TRAVIS_JOB_NUMBER} ${CLIENT_PATH} && ./e2e/integration_tests.sh
     - ./gradlew jacocoTestReport
 after_success:
   - bash <(curl -s https://codecov.io/bash)
+  - If [ "$TRAVIS_BRANCH" == "develop" ]; then
```

Comment:
I believe this refers to a "deployment" procedure. Why is it located not in a `deploy` section?